### DISCUSSION TITLE
ESP32 S2 ADC GPIOs

### DISCUSSION THREAD

**Discussion by martin072** [arendst/Tasmota#16648] [Q&A] [answered]
I have a Lolin S2 mini, and would like to use a CT013. I've connected it to GPIO34 (ADC1-6 according to schematics), however Tasmota doesn't seem to work if I Specify ADC CT for GPIO34. I've noticed the ESP S2 template does not show A0 but IO for this pin. Is this a template issue?

Compiled as  ESPS2, will show this in the module config:

<img width="335" alt="Screenshot 2022-09-25 at 11 28 20" src="https://user-images.githubusercontent.com/24524506/192137106-83471a3e-717d-4c07-bfb5-b8769d56d855.png">

**Comment by sfromis** [comment, marked as answer]
The template is correct about gpio34 not having analog for ESP32-S2, and I don't know where you got that idea from, I certainly do not see such in the schematic for Lolin S2 Mini. Analog pins for ADC1 are gpio 1..10 (ADC2 on gpio 11..20, not usable with Wifi). For whatever reason, Espressif decided on having very different analog pin placements for ESP32 variants.

**Reply by martin072** [reply]
Ah ok, that is my bad.. I looked at the regular ESP32 for the pinout. See what you mean now. Thought it was just the core that's different, but apparently GPIO's also differ. Many thanks!
